R/opendp/README.md

```diff
@@ -0,0 +1,9 @@
+# OpenDP R API
+
+The OpenDP R API largely mirrors the Python API, with some key differences.
+Most Python examples have not yet been translated to R:
+These hints will help you apply the Python examples to R.
+
+- After invoking `library(opendp)` all functions are available: There are no namespaces like those of the Python library.
+- The chaining operator in R is `|>`, as opposed to `>>` in Python.
+- All portions of the Python API that are derived from Rust have parallels in the R API, but packages like `context` that exist only in Python have not yet been implemented in R.
```

Comment:
Can you also mention that type arguments are prefixed with a `.`? Like `T` becomes `.T`.